avaje-jex/src/main/java/io/avaje/jex/StaticFileHandler.java

```diff
@@ -77,8 +77,7 @@ private void sendFile(Context ctx, HttpExchange jdkExchange, String urlPath, Fil
       String mimeType = lookupMime(urlPath);
       ctx.header("Content-Type", mimeType);
       ctx.headers(headers);
-      jdkExchange.sendResponseHeaders(200, canonicalFile.length());
-      fis.transferTo(jdkExchange.getResponseBody());
+      ctx.write(fis);
```

Comment:
I think the jdkExchange.sendResponseHeaders() is missing here / needs to be called.

Reply:
it gets called from within the ctx write method

Reply:
by the BufferedOutputStream